contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -186,7 +186,7 @@ pub fn execute_execute(
     let mut prop = PROPOSALS.load(deps.storage, proposal_id)?;
     // we allow execution even after the proposal "expiration" as long as all vote come in before
     // that point. If it was approved on time, it can be executed any time.
-    if prop.status != Status::Passed {
+    if prop.current_status(&env.block) != Status::Passed {
         return Err(ContractError::WrongExecuteStatus {});
     }
```

Comment:
Might suggest doing:

```
prop.update_status(&env.block);
if prop.status != Status::Passed {
    return Err(ContractError::WrongExecuteStatus {});
}
```

See this as slightly less error prone as checking `prop.status` below will return correct values (if future code ever does this).